lib/spack/llnl/util/tty/__init__.py

```diff
@@ -19,7 +19,19 @@
 
 from llnl.util.tty.color import cprint, cwrite, cescape, clen
 
-_debug = False
+# Debug levels
+#
+# Note that from a standard Python logging level perspective, the following
+# should, perhaps, be descending (from 10).
+DISABLED = 0
+BASIC = 1
+STANDARD = 2
+DETAILED = 3
+LENGTHY = 4
```

Comment:
basic and standard seem... synonymous -- why not combine them for now and have 3 levels?  That would solve @becker33's issue.

Suggest 3 levels:
* `INFO`
* `VERBOSE`
* `DETAILED`

Or if we want to get more magniloquent with it:
* `INFO`
* `YACKETY`
* `PALAVEROUS`

Reply:
> basic and standard seem... synonymous -- why not combine them for now and have 3 levels? That would solve @becker33's issue.
> 
> Suggest 3 levels:
> 
>     * `INFO`
> 
>     * `VERBOSE`
> 
>     * `DETAILED`
> 

I can reduce to 3 given the aforementioned changes.  

My concerns with `INFO` and `VERBOSE` are that we would, in a sense, be overloading the terms with other options that could create some confusion in the user base.   Is that a non-issue?

Reply:
@becker33 should weigh in on this but I don't think it's much of an issue, since these labels are only used internally.

@becker33 and I actually talked about this a bit and I think we agree that we don't need to preserve "what is currently shown to the user" so merging that `BASIC` and `STANDARD` into `INFO` is fine.

Reply:
Eliminated `STANDARD` debug level and changed values for the other more verbose options.  Can make further changes as needed based on responses to my question.

Reply:
I don't love the names `INFO` and `VERBOSE`, but I also would be inclined to stick with integers rather than add descriptive names to them. I think `debug_level = 2` is more descriptive than the descriptive names; the first thing I did when I encountered the descriptive names is go look to see what the list is and what order they're in. Integers solve that problem.

Reply:
Fair enough and consistent with the use of the 'count' argument.

Reply:
I'm ok with integers too.

Reply:
Switched to integers.